src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector256.cs

```diff
@@ -27,7 +27,7 @@ namespace System.Runtime.Intrinsics
     // value instead, thus reducing the number of locals and helping prevent us from hitting
     // the internal inlining limits of the JIT.
 
-    public static class Vector256
+    public static unsafe class Vector256
```

Comment:
What's the significance of this change?

Reply:
It allowed the keyword to be removed from every individual function that used pointers. It has no externally visible semantics.